irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -75,6 +75,21 @@ namespace iroha {
             // batch is successful; release savepoint
             validation_result = true;
             savepoint->release();
+          } else {
+            auto failed_tx_hash = transactions_errors_log.back().tx_hash;
+            for (const auto &tx : batch) {
+              if (tx.hash() != failed_tx_hash) {
+                transactions_errors_log.emplace_back(
+                    validation::TransactionError{
+                        tx.hash(),
+                        validation::CommandError{
+                            "",
+                            1,  // internal error code
```

Comment:
I think, there must be a separate error code for such a reason, as it is hard for client to understand, what happened. Maybe a ToDo for adding new code?